### DISCUSSION TITLE
GUI implementation for faster-whisper

### DISCUSSION THREAD

**Discussion by JuergenFleiss** [SYSTRAN/faster-whisper#503] [Show and tell]
Hi,

we have been using faster-whisper at our university to transcribe interviews and finally ended up building a GUI to enable researchers to transcribe interviews with speaker diarization locally on their computer. 

https://github.com/BANDAS-Center/aTrain

Feel free to have a look, any feedback is welcome.  Faster-whisper was a real asset, as it allowed us to reduce transcription time on CPU, so thanks for all the great work.

Best,
Jürgen

**Comment by dgoryeo** [comment]
Nicely done @Juergen-J-F (and BANDAS)!!! 
I'm just trying to check it out --I used windows app installer. 
Quick question: is there any utility/package that I could use to convert the json file to an srt or vtt file? 
cheers

**Reply by JuergenFleiss** [reply]
Not yet, @dgoryeo, but it's on the list; there is already an issue open on our github repository. Best, Jürgen

**Reply by dgoryeo** [reply]
Thanks @Juergen-J-F . Am I right to think that the json file has the same format as WhisperX? If, so can one use WhisperV write_output to produce srt, vtt?

**Reply by JuergenFleiss** [reply]
@dgoryeo we now added srt as output format in the latest release.
https://github.com/BANDAS-Center/aTrain/releases/tag/v1.1.0

**Reply by dgoryeo** [reply]
Thanks @Juergen-J-F !

I hope someone can continue to take the helm on faster-whisper since @guillaumekln is now tied up with his new job.

**Comment by JuergenFleiss** [comment]
Hi you all, 

we just released v1.2 of our standalone transcription app integrating distilled models and v3-turbo support as well as GUI model downloader and manager. Feedback is always welcome.

https://github.com/JuergenFleiss/aTrain/releases/tag/v1.2.0
